src/server/v2.0/handler/project.go

```diff
@@ -712,14 +715,14 @@ func getProjectMemberSummary(ctx context.Context, p *project.Project, summary *m
 		wg.Add(1)
 		go func(role int, count *int64) {
 			defer wg.Done()
-
-			total, err := pro.GetTotalOfProjectMembers(p.ProjectID, role)
+			ctx = orm.Context()
+			total, err := a.memberMgr.GetTotalOfProjectMembers(ctx, p.ProjectID, role)
```

Comment:
`total, err := a.memberMgr.GetTotalOfProjectMembers(orm.Clone(ctx), p.ProjectID, role)`

orm.Clone(ctx) will insert a new ormer to the ctx so that the request id and other infos still exists in the new ctx.

Reply:
Updated